projects/05-artist-launch/README.pt.md

```diff
@@ -0,0 +1,514 @@
+| 🟣 Visualizar as variáveis categóricas | Visualizar as variáveis categóricas através de gráficos de barras | <ul><li>Você deve criar pelo menos um gráfico de barras para visualizar as variáveis agrupadas anteriormente.</li></ul> | <ul><li>📹 [Como criar gráficos no Looker Studio](https://www.youtube.com/watch?v=B3hPsl-8u3A)</li></ul> |
+| 🟣 Aplicar medidas de tendência central | Calcular medidas de tendência central (Média e Mediana) usando tabelas no Looker Studio ou BigQuery | <ul><li>Você deve calcular medidas de tendência central (média e mediana) para pelo menos uma variável. Um pode analisar `streams` e outro o total de vezes que aparece em uma playlist.</li></ul> | <ul><li>🚨 Altamente recomendado: 📄 [Conceito de estatísticas descritivas](https://docs.google.com/document/d/13xu7cf5LcUN2CE58qwyZoNrhS713Ci0tI9S1odeapig/edit?usp=sharing)</li><li>[🔈 Conceito de estatísticas descritivas (versão em áudio)](https://drive.google.com/file/d/1v6xdtWci7rYtMKibGowl9OxBArT3CVnI/view?usp=drive_link)</li><li>📄 [Conceito de medidas de tendência central](https://docs.google.com/document/d/1Ly_7zLmUjcczqCkQLQwvDYEGyRLtSNQLVi3w8CIh8sI/edit?usp=sharing)</li><li>[🔈 Conceito de medidas de tendência central (versão em áudio)](https://drive.google.com/file/d/1viy63m4n8Gn9aYHNuLJpd8RzFrTNnlw2/view?usp=drive_link)</li><li>📹 [Vídeo sobre tabelas no Looker Studio](https://www.youtube.com/watch?v=7CLCz3PgJ5s)</li><li>💡 Observe o comportamento das variáveis `streams` e presença total em playlists.</li></ul> |
+| 🟣 Calcular correlação entre variáveis | Calcular a correlação no BigQuery usando CORR | <ul><li>Você deve calcular a correlação entre duas variáveis contínuas. Um pode calcular para `streams` e `playlists` e outro para `streams` e `danceability`.</li></ul> | <ul><li>📄 [Conceito de correlação de Pearson](https://docs.google.com/document/d/1HvuENDveavTmlRtSGk3Gv3LqSPNbRPWNYgOU8e3CxTM/edit?usp=sharing)</li><li>📹 [Como calcular correlação no BigQuery](https://www.loom.com/share/9bf3e48cf57d48ca9c81c2c19562dedf?sid=c4c8a611-d0ae-4c4f-a2f5-200ce48746f4)</li><li>💡 Compreender o cálculo de correlação nos ajudará mais tarde na validação das hipóteses.</li></ul> |
+
```

Comment:
Sustituir por:


| Meta | Objetivo | Objetivo Individual | Recurso |
| ---- | ---- | ---- | ---- |
| 🟣 Agrupar dados por variáveis categóricas | Agrupar variáveis categóricas usando tabelas no Looker Studio ou através de GROUP BY no BigQuery | <ul><li>Você deve agrupar pelo menos uma variável categórica, como a quantidade de faixas por artista, a quantidade de faixas por `ano_de_lançamento`, etc.</li></ul> | <ul><li>🚨 **Altamente recomendado**: [Conceito de variáveis categóricas](https://docs.google.com/document/d/1cOTKw04u2008G7JyRaYHzSzyWRjJwxLryn8hIxkO2es/edit?usp=sharing): Neste documento, você encontrará a descrição e exemplos deste conceito. 🔈Versão de áudio: <audio controls><source src="https://raw.githubusercontent.com/Laboratoria/curriculum-data-contenidos/main/PT_P1_segmentacao/TTS/6contenidos_variaveis_categoricas.mp3" type="audio/mp3"></audio></li><li>📹 [Vídeo sobre tipos de variáveis](https://www.youtube.com/watch?v=N7AbQZA06QU&pp=ygUldmFyacOhdmVpcyBjYXRlZ8OzcmljYXMgZSBudW3DqXJpY2FzIA%3D%3D): Neste vídeo você verá uma breve descrição de variáveis ​​categóricas e numéricas.</li><li><ul><li> [Link para acessar o Looker Studio](https://lookerstudio.google.com/navigation/reporting) </li><li> 📹 [Vídeo como começar no Looker Studio](https://www.youtube.com/watch?v=Gx5S8Ry54TU&pp=ygUjY29tbyBjcmlhciB1bSBwYWluZWwgbm8gZGF0YSBzdHVkaW8%3D) </li><li> 📹 [Vídeo Como conectar dados no Looker Studio](https://www.youtube.com/watch?v=dAZ0yfRSKjQ&pp=ygUiY29tbyBjb25lY3RhciBkYWRvcyBubyBkYXRhIHN0dWRpbw%3D%3D) </ li> <li> 📹 [Visão geral Looker Studio](https://www.youtube.com/watch?v=dfAHZ1C_ZQs&list=PLbnAsJ6zlidug4pEvq6tm0h7DPJGkP-aU) </li><li> 📹 [Vídeo como criar um painel/dashboard no Looker Studio](https://www.youtube.com/watch?v=qoKPW5Yn3Dk&pp=ygUjY29tbyBjcmlhciB1bSBwYWluZWwgbm8gZGF0YSBzdHVkaW8%3D) </li><li>📄 [Tutorial do Google Como criar um relatório no Looker Studio](https://support.google.com/looker-studio/answer/06292570?0hl=ES#zippy=%2Csections-of-this-art%C3%ADculo%2Cneste-artigo) </li><li>⚠️O Data Studio mudou recentemente seu nome para Looker Studio, então você pode encontrar referências ao Data Studio em alguns conteúdos na internet. Não se preocupe, é o mesmo produto. </li><ul>📹 Tabelas no Looker Studio](https://www.youtube.com/watch?v=WcxlQG1UdkY)</li><li>💡 Visualize as variáveis categóricas, como o artista, e veja se algum se destaca, use os anos para ver esses dados por ano.</li><li>💡 Experimente fazer isso no BigQuery usando o comando GROUP BY para ver os dados agrupados ou com tabelas no Looker Studio.</li></ul> |
| 🟣 Visualizar as variáveis categóricas | Visualizar as variáveis categóricas através de gráficos de barras | <ul><li>Você deve criar pelo menos um gráfico de barras para visualizar as variáveis agrupadas anteriormente.</li></ul> | <ul><li>📹 [Video como criar gráficos de barras](https://www.youtube.com/watch?v=p4nNJWtStAs)</li></ul> |
| 🟣 Aplicar medidas de tendência central | Calcular medidas de tendência central (Média e Mediana) usando tabelas no Looker Studio ou BigQuery | <ul><li>Você deve calcular medidas de tendência central (média e mediana) para pelo menos uma variável. Um pode analisar `streams` e outro o total de vezes que aparece em uma playlist.</li></ul> | <ul><li>🚨 Altamente recomendado: 📄 [Conceito de estatística descritiva](https://docs.google.com/document/d/12YZ-984bRqWxKGr8M-Yzrq8MjWubBw_i38SEcjDt9tE/edit?usp=sharing). 🔈Versão de áudio: <audio controls><source src="https://raw.githubusercontent.com/Laboratoria/curriculum-data-contenidos/main/PT_P1_segmentacao/TTS/10contenidos_estatistica_descritiva.mp3 " type="audio/mp3"></audio></li><li>📄 [Conceito de medidas de tendência central](https://docs.google.com/document/d/15_eD0wsb1w7GFMmmAm8HPFZJGNzlmsZalMwBke9hSt0/edit?usp=sharing). 🔈Versão de áudio: <audio controls><source src="https://raw.githubusercontent.com/Laboratoria/curriculum-data-contenidos/main/PT_P1_segmentacao/TTS/11contenidos_tendencia_central.mp3 " type="audio/mp3"></audio> </li><li>📹 [Tabelas no Looker Studio](https://www.youtube.com/watch?v=WcxlQG1UdkY)</li><li>💡 Observe o comportamento das variáveis `streams` e presença total em playlists.</li></ul> |
| 🟣 Calcular correlação entre variáveis | Calcular a correlação no BigQuery usando CORR | <ul><li>Você deve calcular a correlação entre duas variáveis contínuas. Um pode calcular para `streams` e `playlists` e outro para `streams` e `danceability`.</li></ul> | <ul><li>📄 [Conceito de correlação de Pearson](https://docs.google.com/document/d/1z_GT13b-GJphDGujDneULn2IOz-KuSmrQpSJBepJF7o/edit?usp=sharing)</li><li> 📹 [Como calcular a correlação no BigQuery ](https://www.loom.com/share/447a64fa6bd5431289a5a9ca0014e8cb?sid=1edc546a-9691-400a-8cde-b63a2f395532)</li><li>💡 Compreender o cálculo de correlação nos ajudará mais tarde na validação das hipóteses.</li></ul> |